frontend/views/containers/navigation/Navigation.vue

```diff
@@ -242,4 +245,11 @@ export default {
 .c-toggle {
   left: 100%;
 }
+
+// Need this nesting because of order of CSS
+.c-navigation .c-toggle .c-toggle-badge {
```

Comment:
I don't see anything that prevent using only .c-toggle-badge selector only on this page.